src/Autosuggest.js

```diff
@@ -156,6 +157,18 @@ class Autosuggest extends Component {
     }
   }
 
+  maybeSelectFirstSuggestion(event, value) {
+    const { selectFirstSuggestion, multiSection, updateFocusedSuggestion } = this.props;
+
+    if (selectFirstSuggestion) {
+      let sectionIndex = multiSection ? 0 : null;
+      let suggestionIndex = 0;
+
+      updateFocusedSuggestion(sectionIndex, suggestionIndex);
+      this.maybeCallOnChange(event, value, 'auto');
```

Comment:
Thinking about this a little more, the auto-select should perform more like a hover than an updown. You probably wouldn't want the input value to change as a soon as it auto-selects the first item--you could never type anything without the input changing out from under you. I think it's still useful to have as it's own type `auto`, but let me know what you think.